pkg/api/latest/latest.go

```diff
@@ -15,13 +15,13 @@ var Version = unversioned.GroupVersion{Group: "", Version: "v1"}
 
 // OldestVersion is the string that represents the oldest server version supported,
 // for client code that wants to hardcode the lowest common denominator.
-var OldestVersion = unversioned.GroupVersion{Group: "", Version: "v1beta3"}
+var OldestVersion = unversioned.GroupVersion{Group: "", Version: "v1"}
```

Comment:
Other than that one place, it's not being used anywhere. Do we need it in that case?